root/components/Paginator.js

```diff
@@ -43,74 +43,74 @@ const Paginator = ({
     <nav>
       <ul className="pagination">
         {pager.previous_page ? (
-          <li>
+          <li key="previous">
             <a href={uriWith(reqUri, {page: pager.previous_page})}>
               {l('Previous')}
             </a>
           </li>
         ) : (
-          <li>
+          <li key="no-previous">
             <span>{l('Previous')}</span>
           </li>
         )}
 
-        <li className="separator" />
+        <li className="separator" key="separate-previous" />
 
         {start > pager.first_page ? (
-          <li>
+          <li key="first">
             <a href={uriWith(reqUri, {page: pager.first_page})}>
               {pager.first_page}
             </a>
           </li>
         ) : null}
 
         {start > (pager.first_page + 1) ? (
-          <li>
+          <li key="after-first">
             <span>{l('…')}</span>
           </li>
         ) : null}
 
         {range(start, end + 1).map(page => (
           (pager.current_page === page) ? (
-            <li>
+            <li key={"current-" + page}>
```

Comment:
I would use `"number-" + page` here too since it can patch the existing element if it exists.

Reply:
Replaced in https://github.com/metabrainz/musicbrainz-server/pull/790/commits/9c8a90a688a2fda95ed746bc4e7b36d0137e6c0b.